### ISSUE TITLE
Email: Comment: WCAG 3.0 - Issue: Accessibility to Web Content Requiring a CAPCHA for Access

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#475]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Mar/0003.html): 

From: Rod Macdonald (rjmacdonald@hawaiiantel.net)

Re: W3C Accessibility Guidelines (WCAG) 3.0 - W3C First Public Working Draft 21 January 2021

Issue: Accessibility to Web Content Requiring a CAPCHA for Access

Background: For purposes of this discussion, individuals who are Deaf-Blind can be grouped into four sub-groups:

(1) The Deaf-Blind individual retains sufficient residual hearing, with amplification and/or other enhancements, to access web content in the of a hearing person.

(2) The Deaf-Blind individual retains sufficient residual vision, with screen magnification and/or other enhancements, to access web content in the manner of a sighted person.

(3) The Deaf-Blind individual cannot access web content via speech or hearing, but can do so using braille.

(4) The Deaf-Blind individual cannot access web content using vision, hearing or braille, and thus cannot access web content at all. (There may be extremely rare cases when the use of unusual technology may circumvent this.)

This discussion refers exclusively to Deaf-Blind individuals in the third group - braille users.

Problem: Numerous providers of online services use a technique called a "CAPCHA" (Google calls theirs a "RECAPCHA") to ensure that customers are in fact human. Highly stylized characters are displayed on the screen and the user is asked to type the characters on the keyboard. Vision is required to do this. Typically, an audio work-around is provided so that blind individuals can gain access. However, there is no known workaround if the consumer does not have the vision OR the hearing to gain access. A consumer using a screen reader and braille display is completely excluded. 

Comments: There is no known solution to this accessibility issue.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
Proposed response:

WCAG 2.2 included WCAG 3.3.8 Accessible Authentication.  While WCAG 3 is not constrained to be backward compatible, WCAG 3 is expected to not lose requirements.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
